pkg/fluxexec/bootstrap_github_test.go

```diff
@@ -0,0 +1,104 @@
+package fluxexec
+
+import (
+	"context"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("bootstrapGitHubCmd", func() {
+	It("should be able to generate correct install commands", func() {
+		By("generating the default command", func() {
+			path, err := exec.LookPath("flux")
```

Comment:
There's some more `exec.LookPath` - could you change the rest, too?

Reply:
Ha. Sorry about that @ozamosi 
I removed all of those lookups.